src/fallbackError.ts

```diff
@@ -75,4 +76,18 @@ export class FallbackErrorDecoder {
     };
     return result;
   }
+
+  // Construct an Error from a StatusObject.
+  // Adapted from https://github.com/grpc/grpc-node/blob/master/packages/grpc-js/src/call.ts#L79
+  callErrorFromStatus(status: DecodedRpcStatus): Error {
+    const message = `${status.code} ${Status[status.code]}: ${status.message}`;
+    delete status.message;
```

Comment:
Why delete the message from the object?

Reply:
In line 85, when I do `new Error(message)` it sets the message, but `Object.assign` will overwrite it from `status.message`. I can either delete it, or assign `status.message = message` (I can change it to the latter if you think it's more readable).

Reply:
Changed this to more readable code.